docs/index.md

```diff
@@ -1,148 +1,128 @@
-| timestampResolved | Time the incident was resolved            | Number (UNIX epoch timestamp in seconds) |
+-  [Deployments](#deployments)
+-  [Changes](#changes)
+-  [Incidents](#incidents)
 
-```sh
-./event-cli push incident \
-    --api-key "<API-KEY>" \
-    --identifier "<incident identifier>" \
-    --timestampCreated "<UNIX epoch timestamp in seconds>" \
-    --timestampResolved "<UNIX epoch timestamp in seconds>"
-```
+!!! important
+    Before setting up the integration with Pulse, make sure that you have an API key that identifies your organization and authorizes you to send data to Pulse.
 
-!!! tip
-    You can get the UNIX epoch timestamp in seconds with `date +%s`.
+#### Deployments
 
-### HTTP API
+**For SaaS applications,** send information to Pulse whenever you deploy to production.
 
-Since it might be difficult to use the CLI to send some types of events (like changes or incidents),
-we also support pushing events to an HTTP POST endpoint.
+**For self-hosted applications,** a better option might be to send information to Pulse whenever you have a valid artifact ready to be delivered to any user.
 
-This might be useful for sending data from providers that only support webhooks. You can set up your provider to send a webhook to the following URL:
-`https://ingestion.acceleratedevops.net/v1/ingestion/<provider>?api_key=<API-KEY>`
+| Field      | Description                                            | Format                                       |
+| ---------- | ------------------------------------------------------ | -------------------------------------------- |
+| identifier | Version or another unique identifier of the deployment | String                                       |
+| timestamp  | Time when the deployment finished                      | Number<br/>(Unix epoch timestamp in seconds) |
+|            | Commit identifiers included in the deployment          | String<br/>(space-separated list)            |
 
-There is no specified format for events sent to this endpoint. Although, make sure to include all the fields documented above. If you are planning to push events this way, please let us know. Data will not be immediately available in your dashboard as we'll need to process it on our side.
+```sh
+./event-cli push deployment \
+    --api-key "<API KEY>" \
+    --identifier "<deployment identifier>" \
+    --timestamp "$(date +%s)" \
+    <space-separated list of commit identifiers>
+```
 
-## Examples
-Some examples that we used to quickly populate our dashboard with historical data.
+<!-- IDEA
+     Consider including example snippets for the webhook -->
 
-### Using the CLI to push historic incidents
-In this example we used the CLI to push past incidents from the Codacy status page API into Pulse.
+#### Changes
 
-```py
-# incidents.py
+Send information to Pulse whenever a commit is pushed to a repository.
```

Comment:
Before, this line mentioned `A commit pushed to a Git`. Confirm that indeed the actual CVS does not matter.

Reply:
yup!
based on the data model we are using, revisions from any version control system can be used.

a nice thing to highlight: when some people time about measuring lead time for changes they'll can think about measuring from the issue tracker. it might we worth to clarify - in the metric page? - that the metric is only measured from the moment code is checked in, and therefore `changes` should only concern version control system events.

Reply:
Yes, it makes sense to clarify the metric. I'm adding a short sentence to the **Lead time for changes** metric.